ecommerce/models.py

```diff
@@ -28,6 +28,7 @@
 from django.db.models.fields.related import (
     ForeignKey,
 )
+from django.db.models.query_utils import Q
```

Comment:
nit: I think that the [suggested way](https://docs.djangoproject.com/en/1.10/topics/db/queries/#complex-lookups-with-q-objects) to make this import is 
`from django.db.models import Q`